name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/views/dashboard/heatmap/PerformanceHeatmapWidget.java

```diff
@@ -87,18 +89,19 @@ public PerformanceHeatmapWidget(Widget widget, DashboardData data)
             HeatmapModel.Row<Double> row = new HeatmapModel.Row<>(label);
 
             // monthly data
-            for (LocalDate month = LocalDate.of(year.getValue(), 1, 1); month.getYear() == year
-                            .getValue(); month = month.plusMonths(1))
+            for (YearMonth month = YearMonth.of(year.getValue(), 1);
+                 month.getYear() == year.getValue();
+                 month = month.plusMonths(1))
             {
-                if (actualInterval.intersects(Interval.of(month, month.plusMonths(1).minusDays(1))))
+                if (actualInterval.intersects(Interval.of(month.atDay(1).minusDays(1), month.atEndOfMonth())))
```

Comment:
The `.minusDays(1)` here is the bug fix, changing the starting date from 2021-03-01 to 2021-02-28.